test/unit/learning-catalogue/model/dateRangeValidatorTest.ts

```diff
@@ -0,0 +1,86 @@
+import {DateRange} from '../../../../src/learning-catalogue/model/dateRange'
+import {Validator} from '../../../../src/learning-catalogue/validator/validator'
+import {DateRangeFactory} from '../../../../src/learning-catalogue/model/factory/dateRangeFactory'
+import {Factory} from '../../../../src/learning-catalogue/model/factory/factory'
+import * as moment from 'moment'
+import {expect} from 'chai'
+
+describe('DateRange validation tests', () => {
+	let dateRangeFactory: Factory<DateRange>
+	let validator: Validator<DateRange>
+
+	before(() => {
+		dateRangeFactory = new DateRangeFactory()
+		validator = new Validator<DateRange>(dateRangeFactory)
+	})
+
+	it('should pass validation if valid', async () => {
```

Comment:
NAMING - what makes a `valid` date? IE: `should validate if date is in the future on a Saturday`